src/components/SearchBar.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+import './SearchBar.css';
+
+export default function SearchBar() {
+  const [search, setSearch] = useState('');
+
+  const handleChange = (e) => {
+    setSearch(e.target.value);
+  };
+
+  return (
+    <input
+      className='search'
+      type='text'
+      value={search}
+      onChange={handleChange}
+      placeholder='🔍 링크를 검색해 보세요.'
+    ></input>
```

Comment:
```suggestion
    <input
      className='search'
      type='text'
      value={search}
      onChange={(e)=>{setSearch(e.target.value}}
      placeholder='🔍 링크를 검색해 보세요.'
    />
```

개인적으로는 이벤트 자리에 작성하는 코드가 한줄이라면 위와 같이 작성하는 것을 선호합니다!
그리고 input 태그는 selfclosing 해주세요!